controller/dataplane/owned_deployment.go

```diff
@@ -27,6 +29,8 @@ import (
 	operatorv1beta1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1beta1"
 )
 
+const restartAnnotationKey = "kubectl.kubernetes.io/restartedAt"
```

Comment:
Can we add a comment to this `const`? Possibly linking it to a relevant k8s doc page?